File: C/lt_01_estseq/lt01_estseq_27.c
#include <stdio.h>

int main() {

  double spaco, temp;

  printf("Insira a quantidade de voltas: ");
  scanf("%lf", &spaco);

  printf("Insira a extensão do percurso: ");
  scanf("%lf", &temp);

  spaco = (spaco * temp) / 1000;

  printf("Insira o valhor do tempo: ");
  scanf("%lf", &temp);

  temp /= 60;

  spaco = (spaco / temp);

  printf("A velocidade media foi %.2lf Km/h.\n", spaco);
  return 0;
}

File: Java/LT01_ESTSEQ/LT01_ESTSEQ_27.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */

/**
 * Nome do Programador: Lucas Sartorelli
 * Data: 22/02/2025
 * Descrição:
 * Receba o número de voltas, a extensão do circuito 
 * (em metros) e o tempo de duração (minutos). 
 * Calcule e mostre a velocidade média em km/h.]]
 */
import javax.swing.JOptionPane;
public class LT01_ESTSEQ_27 {
    public static void main(String args[]){
        Double vol, circ, temp;
        circ = Double.parseDouble(JOptionPane.showInputDialog("Insira a extensão do Circuito."));
        vol = Double.parseDouble(JOptionPane.showInputDialog("Insira o númeor de voltas."));
        temp = Double.parseDouble(JOptionPane.showInputDialog("Insira o Tempo de Duração"));
        vol = (((vol*circ)/temp)*3.6);
        JOptionPane.showMessageDialog(null,"A velocidade média é " +vol +"km/h.");
    }
}
